news/2018-09-18-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+
+### Beatmap Nominator Retirements
+
+Fun times always have to come to an end eventually. This month, we give our farewells to two of our diligent Beatmap Nominators: [Plaudible](https://osu.ppy.sh/users/7149815) and [Jonawaga](https://osu.ppy.sh/users/3653035).
+
+On behalf of the community, we thank you for your hard work in bringing new content into the game, and we wish you good luck in your next steps in life!
+
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+### Ongoing discussions
+
+Normally, every monthly issue of the QAT Gazette brings you exciting news and updates on a great diversity of fields within the osu! Mapping and modding community.
+
+Unfortunately, this month is somewhat sparse on the news front, since some important topics require further discussion and have not yet been resolved. However, we are writing this in order to inform you that we indeed have not simply forgotten about such matters!
+The topics still being debated include the following:
+
++ Discussion about the [management of hybrid mapsets](https://osu.ppy.sh/community/forums/topics/792131) 
++ Discussion about [two sets of the same song being allowed in the Qualified beatmap section](https://osu.ppy.sh/community/forums/topics/787252)
++ Status of the feature request list we informed about in the [last month’s Gazette](https://osu.ppy.sh/home/news/2018-09-19-the-qat-gazette-august-2018)
+
+### New Quality Assurance Team additions
+
+As you may have already guessed from this article, we are now graced to have the presence of [Kibbleru](https://osu.ppy.sh/users/3193504) within our ranks! After two months in the former Beatmap Appreciation Team, followed by a further three years of experience in the Beatmap Nomination Group, he has made the next step at last and will join the QAT, assisting the osu! division in their efforts to maintain high levels of beatmap quality and Nominator proficiency.
+
+Lest we forget about [-Kazu-](https://osu.ppy.sh/users/920861) also! With seven months in the Beatmap Nominators and seven *years* of experience in osu!taiko, he too has taken the plunge in order to share his vast experience in assisting with disqualifications and Beatmap Nominator applications to support the ever increasing size of the osu!taiko mapping scene.
+
+### Quality Assurance Team retirements
+
+Whilst we may have welcomed two new members, we must also sadly bid farewell to two others. [Hobbes2](https://osu.ppy.sh/users/8157492) has stepped down from his post within the QAT to return to the more hands-on aspects of modding as a Beatmap Nominator, whilst his fellow osu! compatriot [Mir](https://osu.ppy.sh/users/8688812) has elected to enjoy a period of rest as an osu! Alumni. 
```

Comment:
I wouldn't say whilst

and you wrote it twice heeehhh

Reply:
Changed it to "whereas"